### ISSUE TITLE
profile: Include requested admin name for records pending and approved

### ISSUE THREAD

**Issue by dannypernik** [opencurrents/opencurrents#678]
@nickolashe are there variables available for showing the first and last name of the admin who the volunteer requested to approve hours? We want the text of requested/approved hours on profile to say "You requested X hours from (admin first/last) at (orgname)"

I tried understanding how to access AdminActionUserTime from the record but not quite getting it. If this requires additional work to access the admin name, we can move it out of MVP, but if it's available then I can make the change if you let me know the appropriate variable names for admin first/last.

**Comment by nickolashe** [comment]
@dannypernik good point. to answer your question, `record` **is** an instance of `AdminActionUserTime` in this case. admin user can be accessed using `record.user`

**Comment by nickolashe** [comment]
https://github.com/opencurrents/opencurrents/blob/develop/openCurrents/templates/user-hour-request-list.html
